docs/photoniq/vwrs/vwrs-best-practices.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 110
+title: VWRs Best Practices
+---
+
```

Comment:
These best practices, are they for all VWRs?  If so, the two practices below are specific to native apps.  They don't apply to web-bases apps.

Reply:
Yes, it is best practices for all VWRs. If you can give me a few for web apps, then I can make two sections. Otherwise, I'll just specify these two are for mobile applicaitons.